README.md

```diff
@@ -115,11 +115,13 @@ between client-python versions.
 | 8.0 Alpha/Beta  | Kubernetes main repo, 1.12 branch    | ✗                             |
 | 8.0             | Kubernetes main repo, 1.12 branch    | ✗                             |
 | 9.0 Alpha/Beta  | Kubernetes main repo, 1.13 branch    | ✗                             |
-| 9.0             | Kubernetes main repo, 1.13 branch    | ✓                             |
+| 9.0             | Kubernetes main repo, 1.13 branch    | ✗                             |
 | 10.0 Alpha/Beta | Kubernetes main repo, 1.14 branch    | ✗                             |
 | 10.0            | Kubernetes main repo, 1.14 branch    | ✓                             |
 | 11.0 Alpha/Beta | Kubernetes main repo, 1.15 branch    | ✗                             |
 | 11.0            | Kubernetes main repo, 1.15 branch    | ✓                             |
+| 12.0 Alpha/Beta | Kubernetes main repo, 1.16 branch    | ✗                             |
+| 12.0            | Kubernetes main repo, 1.16 branch    | ✓                             |
```

Comment:
```suggestion
| 12.0 Alpha/Beta | Kubernetes main repo, 1.16 branch    | ✓                             |
```

Since, this release involves an Alpha client, we should add a tick against that row. When the stable client releases, we will add the last line and cross out the alpha/beta support.